include/zephyr/bluetooth/hci_types.h

```diff
@@ -2905,6 +2934,26 @@ struct bt_hci_evt_le_req_peer_sca_complete {
 	uint8_t  sca;
 } __packed;
 
+/** Reason for Transmit power reporting.
+ */
+/* Local Transmit power changed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_LOCAL_CHANGED         0x00
+/* Remote Transmit power changed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_REMOTE_CHANGED        0x01
+/* HCI_LE_Read_Remote_Transmit_Power_Level command completed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_READ_REMOTE_COMPLETED 0x02
+
+#define BT_HCI_EVT_LE_TRANSMIT_POWER_REPORT     0x21
+struct bt_hci_evt_le_transmit_power_report {
+	uint8_t	 status;
+	uint16_t handle;
+	uint8_t  reason;
+	uint8_t  phy;
+	int8_t   tx_power_level;
+	int8_t   tx_power_level_flag;
```

Comment:
This shouldn't be signed, so use `uint8_t`. The power level and delta can be negative, but not the flags.